packages/mobile/README.md

```diff
@@ -94,7 +94,7 @@ _If you are a Valora employee, please ask to be added to the Valora iOS developm
 
 Xcode is needed to build and deploy the mobile wallet to your iOS device. If you do not have an iOS device, Xcode can be used to emulate one.
 
-Install [Xcode 12.2](https://developer.apple.com/download/more/?q=xcode) (an Apple Developer Account is needed to access this link).
+Install [Xcode 13.2](https://developer.apple.com/download/more/?q=xcode) (an Apple Developer Account is needed to access this link).
```

Comment:
Maybe we can just have `Xcode 13` here? I am using 13.3 now and it seems to work okay, so not sure that there's much reason to also specify the minor version here?

Reply:
Fixed in https://github.com/valora-inc/wallet/pull/2381/commits/92a6d9afafabfbd72bb588ff4cd66ffec4fce54d!